resources/views/pages/services-list.blade.php

```diff
@@ -52,7 +52,11 @@
                                     <form action="{{route("admin.services.destroy",$service)}}" method="post" class="mb-0">
                                         @csrf
                                         @method("DELETE")
+<<<<<<< HEAD
                                         <button type="submit" class="btn-link border-0 bg-white" 
+=======
+                                        <button type="submit" class="btn btn-link"
+>>>>>>> d4bc8c08d76695cabfb90ae11e0167ae85fb7e2b
```

Comment:
Is this changed fixed?

Reply:
It is already been fixed since the last time I make pull request.